types/react-syntax-highlighter/index.d.ts

```diff
@@ -44,15 +66,15 @@ declare module "react-syntax-highlighter" {
 // esm start
 declare module "react-syntax-highlighter/dist/esm/default-highlight" {
     import * as React from "react";
-    import { SyntaxHighlighterProps } from 'react-syntax-highlighter';
+    import { SyntaxHighlighterProps } from "react-syntax-highlighter";
     export default class SyntaxHighlighter extends React.Component<
```

Comment:
In 13.0 these components are now function components (not very important other than forbidding refs)